Add note on NavigateToString limit and common error (#3236)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/working-with-local-content.md

Loading local content by using the `NavigateToString` method doesn't make it possible for the resulting document to reference additional web resources such as CSS, image, or script files. The method only lets you specify the string content of the HTML document. To reference additional web resources from your HTML document, use one of the other approaches described in this article, or represent those additional web resources inline in the HTML document.

> ###### Size Limit

Reviewer: revised as:
```suggestion
###### Size limit
```

> `NavigateToString` has a size limit of 2MB, which combined with needing to inline additional resources may be easy to exceed. If you do, you'll hit a "Value does not fall within the expected range" error.

Reviewer: revised as:
```suggestion
`NavigateToString` has a size limit of 2MB.  This size limit may be easy to exceed, when combined with inlined additional resources.  If this size limit is exceeded, an error is returned: "Value does not fall within the expected range".
```

Reviewer: when what is combined with inlined stuff?  need wording @aluhrs13 eg:
should be worded like: 
easy to exceed, when the basic url is combined with inlined additional resources.
